### DISCUSSION TITLE
Disable onboarding for our organization

### DISCUSSION THREAD

**Discussion by pietrodn** [renovatebot/renovate#18826] [Request Help] [answered]
Hi,

I saw that you're open to disabling the onboarding PR per organisation, as in https://github.com/renovatebot/renovate/discussions/18432. Would you mind applying the same change to our organisation, at https://github.com/BendingSpoons?

We don't want Renovate to run on any repo, unless we push a configuration file in there.

Thank you!

**Comment by rarkins** [comment, marked as answer]
I've set onboarding=false for your org, please double check it with a single repo first to make sure there's no onboarding as expected

**Comment by davidepedranz** [comment]
Hi,

Would it be possible to disable the onboarding PR also for our organization https://github.com/Evernote?

Thanks a lot in advance 🙏🏼

**Reply by rarkins** [reply]
Do you want Renovate to run on all repos (config or no config), or just run on repos which have manually pushed a config file?

**Reply by davidepedranz** [reply]
If possible, only on repos with a config file 🙏🏼

**Reply by rarkins** [reply]
Sorry for the late reply. This would mean the default config for your org would simply be:

```
{
  "onboarding": false
}
```

`requireConfig` defaults to `required` already.

But do you also want a default preset, either your own, or `"config:base"`? e.g.

```
{
  "extends": ["config:base"],
  "onboarding": false
}
```

**Reply by davidepedranz** [reply]
```
{
  "onboarding": false
}
```

This one should work. We already have internal base configurations: there is no need to extend `config:base`.

Thanks a lot for your help 🙏🏼

**Reply by rarkins** [reply]
So this is the behavior you'll see:
- If a user hasn't pushed a repo config then Renovate will skip the repo
- If a user has pushed a repo config but forgot to extend your internal base config, then you'll get default Renovate behavior
- If a user pushes a repo config extending your internal config, you'll get expected behavior

Please confirm if that's ok, and I'll make the settings

**Reply by rarkins** [reply]
Also, I'm not seeing github.com/Evernote installed yet. If it's not yet installed, please pick a single repo (e.g. your shared preset one) and "Select repositories" just that one at time of installation. Once it's installed, I can add default config to it.

**Reply by davidepedranz** [reply]
Hi @rarkins, sorry for the delay.

We have installed Renovate on the organization. Unfortunatelly, there was a small internal misunderstanding, and we enabled it for all repositories. I hope it is not a problem (we will delete the PRs manually).

Can you please add the configuration we discussed above for the organization? 🙏🏼

**Reply by rarkins** [reply]
@davidepedranz I've set onboarding=false for this org in our backend now

I think we can push a small fix to Renovate so that any remaining onboarding PRs will be automatically cleaned up in the next day or two. i.e. if onboarding=false but an onboarding PR exists then autoclose it

**Reply by rarkins** [reply]
Relevant PR: #21967

Once this is live in the app, the accidental onboarding PRs should all be autoclosed the next time Renovate runs on the repo

**Reply by pietrodn** [reply]
I already closed them all through this one-liner: `gh search prs --owner Evernote --match title --app renovate --state open --limit=100 --json url "Configure Renovate" | jq '.[].url' | xargs -n 1 gh pr close`

Thanks!

**Reply by rarkins** [reply]
That's a great command to know, thanks

**Reply by davidepedranz** [reply]
Amazing, thanks a lot for your help @rarkins 🙏🏼 And great idea the PR to automatically close onboarding PRs: I am sure it will be useful for other organizations in the future.
